op-acceptance-tests/acceptance-tests.yaml

```diff
@@ -2,14 +2,37 @@
 #
 # All acceptance tests need to be registered here for op-acceptor to run them.
 #
-# Note: The current acceptance tests are placeholders; real ones to come soon.
+# Note: The 'common' acceptance tests are placeholders; real ones to come soon.
 
 gates:
-  - id: localnet
-    description: "Localnet validation gate"
+  - id: common
+    description: "Common tests for all networks."
     tests:
       - name: TestFindRPCEndpoints
-        package: github.com/ethereum-optimism/optimism/kurtosis-devnet/pkg/kurtosis/api/run
+        package: github.com/ethereum-optimism/optimism/kurtosis-devnet/pkg/kurtosis
+
+  - id: mainnet
+    inherits:
+      - common
+    description: "Mainnet network tests."
+    tests:
       - package: github.com/ethereum-optimism/optimism/op-acceptance-tests/tests/interop
       - package: github.com/ethereum-optimism/optimism/op-acceptance-tests/tests/isthmus
       - package: github.com/ethereum-optimism/optimism/op-acceptance-tests/tests/fjord
+
+  - id: isthmus
+    inherits:
+      - common
+      - mainnet
+    description: "Isthmus network tests."
+    tests:
+      - package: github.com/ethereum-optimism/optimism/op-acceptance-tests/tests/isthmus
+
+  - id: interop
+    inherits:
+      - common
+      - mainnet
+    description: "Interop network tests."
+    tests:
+      - package: github.com/ethereum-optimism/optimism/op-acceptance-tests/tests/interop
+      - package: github.com/ethereum-optimism/optimism/op-acceptance-tests/tests/fjord
```

Comment:
I'd argue for an organization around one gate per release. Under that approach this gate (interop) would inherit from another fjord gate rather than specifying a direct dependency.